### PR TITLE
Feature/update button event info sheet

### PR DESCRIPTION
**Pull request by Greenade**
created a button to edit the event on the event info sheet, updated the viewmodel accordingly
image

### CODE REVIEW

File: app/src/main/java/com/github/swent/echo/compose/components/EventInfoSheet.kt [hunk 1]
```diff
@@ -108,7 +110,14 @@ fun EventInfoSheet(
                 style = MaterialTheme.typography.bodyLarge
             )
 
-            Row(modifier = Modifier.align(Alignment.BottomCenter).padding(bottom = 70.dp)) {
+            val rowModifier =
```
Comment (ntissieres) [inline app/src/main/java/com/github/swent/echo/compose/components/EventInfoSheet.kt:113]: can be a one-liner : Modifier.align(Alignment.BottomCenter).padding(bottom = 70.dp, end = if(canModifyEvent) 140.dp else 0.dp)

**Review by ntissieres — CHANGES_REQUESTED** [review]
A minor change can be done to avoid unnecessary code but the rest is fine, good work !

File: app/src/main/java/com/github/swent/echo/compose/components/EventInfoSheet.kt (hunk 1 above)
Comment (Greenade) [reply app/src/main/java/com/github/swent/echo/compose/components/EventInfoSheet.kt:113]: makes sense, didn't think about that

**Review by ntissieres — APPROVED** [review]
Perfect !

**Pull request merged**